docs/source/contrib.rst

```diff
@@ -0,0 +1,44 @@
+Dask Contrib
+============
+
+In addition to the primary `Dask GitHub Organization <https://github.com/dask>`_ there is also a secondary
+`Dask Contrib GitHub Organization <https://github.com/dask-contrib/>`_ which contains community supported
+projects which integrate with or enhance the Dask experience.
+
+These projects are owned and maintained by folks outside of the core Dask maintainer group and therefore
+the quality and stability may be different to that of the main Dask packages. However these projects are
+expected to conform to the `Dask Code of Conduct <https://github.com/dask/governance/blob/main/code-of-conduct.md>`_.
+
+The contrib organisztion exists to improve discoverability of user submitted projects.
+
+Contrib package criteria
+------------------------
+
+If you are working on a project which you think could be a great addition to the Dask Contrib org then please
+feel free to raise an issue on the `Dask Community repo <https://github.com/dask/community>`_ requesting a transfer.
+
+For a project to be accepted into Dask Contrib it must meet the following criteria:
+
+- Packages must complement or enhance Dask in some way, maybe it contains plugins, adds integration with some other library or just generally improves the Dask experience for users.
+- Packages must have at least one active maintainer.
+- Packages must be installable both from PyPI with ``pip`` and Conda Forge with ``conda``.
+- Packages must have documentation.
+
+Graduation to the main org
+--------------------------
+
+In some cases it may be appropriate to graduate a project to the main Dask Organization, or perhaps merge the code/functionality
+into an existing Dask project.
+
+This will be assessed on a case-by-case basis by the Dask maintenance team.
+
+For some projects it may make sense to graduate their content into other repositories. The now archived ``dask-lightgbm`` project was integrated into
+LightGBM for example.
+
+Generally for a project to make it into the main Dask Org it should meet the following criteria:
+
+- Packages much have an active community.
+- Packages must be maintained by one or more people who are employed to do so and will dedicate a minimum of 1 day per week to Dask.
```

Comment:
I'm a little conflicted over the "who are employed to do so" part of this criteria. On the one hand I think that if project maintenance is a part of one's job it's much more likely that the maintainer will stay on top of things. On the other hand there are lots of OSS volunteers who might also dedicate 1 day per week towards project maintenance.

I'm not necessarily saying we should remove the employment aspect of this criteria, but I did want to at least raise my concern about it

Reply:
Yeah I appreciate what you're saying.

My goal here was to indicate that there should be some firm commitment from a maintainer, not just a casual side project.

Reply:
Yeah I think we can scratch "who are employed to do so" it ran a little odd to my ears as well.

Reply:
I like @jacobtomlinson's idea of a "firm commitment" -- that seems to communicate the core message that projects in the Dask org shouldn't be treated as casual side projects

Reply:
Are these standards evaluated continually or only on admission? Should there be a demotion policy?

I'm thinking of libraries like [dask-tensorflow][1] that satisfied most of these requirements during development. Currently, it does not satisfy any requirements for a main Dask org package and barely satisfies requirements for a Dask-contrib package.

[1]:https://github.com/dask/dask-tensorflow
[2]:https://github.com/dask/dask-lightgbm

Reply:
That's a great question @stsievert. As of today we've never removed a project from the Dask GitHub org -- though recently we've archived several repos that were no longer maintained (e.g. dask-tensorflow is archived). That said, I get the sense there's an appetite for being able to move some projects from the main dask org into dask-contrib so we can keep them around while (trying to) set user expectations around project stability and maintenance. 

In practice I suspect we'll want to handle this on a case-by-base basis much like how we plan to move projects from dask-contrib into dask. Perhaps a lightweight sentence or two about opening up an issue in the Dask community tracker for moving a project from dask-contrib -> dask would be sufficient.

Reply:
> while (trying to) set user expectations around project stability and maintenance.

That'd make a *great* documentation page. That page would help make the (experimental) integrations a lot more visible to users, and especially if it points to projects that aren't in dask-contrib but make heavy use of Dask (like dask-lightgbm).x

There should also probably be a section on the different main Dask org libraries integrate (e.g., https://ml.dask.org/pytorch.html).

Reply:
I think this line is the only bit that needs to be changed (dropping the employment criteria in favor of something a bit softer). @jacobtomlinson do you have time to finish this up?